fake-toolchain/Unix/cc

```diff
@@ -9,6 +9,6 @@ elif [ "$1" = "-v" ] || [ "$1" = "--version" ]; then
     $YCM_CONFIG_GEN_CC_PASSTHROUGH $@
 
 else
-    echo "$@" >> $YCM_CONFIG_GEN_CC_LOG
+	echo "YCM_CONFIG_GEN_PWD=$(pwd) $@" >> $YCM_CONFIG_GEN_CC_LOG
```

Comment:
On second thought, while having spaces in a path _does_ break this commit, I don't think adding quotes is the fix.  `YCM_CONFIG_GEN_PWD` is not a shell variable.  `YCM_CONFIG_GEN_PWD=<path>` is just a special word that gets created and parsed [here](https://github.com/rdnetto/YCM-Generator/blob/stable/config_gen.py#L379-L381).  This is where the issue is, because there is no support for spaces in words :)

In addition to breaking this commit, I suspect that this limitation of words also prevents proper parsing of [filename_flags](https://github.com/rdnetto/YCM-Generator/blob/stable/config_gen.py#L370) with spaces in the path such as `-I "a directory with spaces"`.